src/js/components/Header/Header.js

```diff
@@ -1,19 +1,141 @@
-import React from 'react';
-
+import React, { useContext, useEffect, useState } from 'react';
+import { ThemeContext } from 'styled-components';
+import { PageContext } from '../Page';
+import { useForwardedRef } from '../../utils';
 import { Box } from '../Box';
 
-const Header = React.forwardRef(({ ...rest }, ref) => (
-  <Box
-    align="center"
-    as="header"
-    direction="row"
-    flex={false}
-    justify="between"
-    gap="medium"
-    {...rest}
-    ref={ref}
-  />
-));
+const Header = React.forwardRef(({ sticky, background, ...rest }, ref) => {
```

Comment:
Wondering if the signature of `sticky` should be:

sticky: 'scrollup' | { type: 'scrollup', fill: 'horizontal' }